gen/calc.py

```diff
@@ -1217,12 +1248,15 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'auth_cookie_secure_flag': 'false',
         'marathon_java_args': '',
         'master_dns_bindall': 'true',
+        'mesos_cgroup_enable_cfs': 'true',
         'mesos_dns_ip_sources': '["host", "netinfo"]',
         'mesos_dns_set_truncate_bit': 'true',
         'master_external_loadbalancer': '',
         'mesos_log_retention_mb': '4000',
+        'mesos_logrotate_file_size_mb': '2',
         'mesos_container_log_sink': 'fluentbit+logrotate',
-        'mesos_max_completed_tasks_per_framework': '',
+        'mesos_max_completed_frameworks': '10',
```

Comment:
This is changing the default as well right (previous was 50)? I didn't see this captured in the PR description.

Reply:
I personally would prefer changing the defaults in a separate PR than this one which is about making some flags configurable.

Reply:
Updated the PR description about the default change. Putting them together since it is all about DCOS-59021.